src/helpers.js

```diff
@@ -100,3 +108,30 @@ export function runQuery (instance, query, needResponse) {
     return response
   })
 }
+
+export function findLastObject (instance, table) {
+  return new Promise((resolve, reject) => {
+    let queryInfo = `PRAGMA table_info("${table}")`
+    let queryId = `SELECT seq FROM sqlite_sequence WHERE name="${table}"`
```

Comment:
All the columns could be the parameters - we just need to return one that meets all of them. If there's not a unique or primary key that's the best we can do.

The only thing we really need to go out of our way here for I think is autoincrements, because we don't know what that is until it's been inserted.